modules/sharedaddy/sharing.php

```diff
@@ -173,6 +180,10 @@ public function output_service( $id, $service, $show_dropdown = false ) {
 <?php
 	}
 
+	public function wrapper_admin_page() {
+		Jetpack_Admin_Page::wrap_ui( array( &$this, 'management_page' ), array( 'is-wide' =>true ) );
```

Comment:
`Jetpack_Admin_Page::wrap_ui` is only available in Jetpack - this file cannot be synced back as-is

Reply:
Would it make sense to add this on the WP.com side since additional wp-admin pages in the future will likely adopt using this function?